contrib/exporters/secadvisor/mod/mangle_logstatus_test.go

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2019 IBM, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy ofthe License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specificlanguage governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package mod
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/spf13/viper"
+
+	awsflowlogs "github.com/skydive-project/skydive/contrib/exporters/awsflowlogs/mod"
+	"github.com/skydive-project/skydive/contrib/exporters/core"
+	"github.com/skydive-project/skydive/flow"
+)
+
+var logStatusConfig = []byte(`---
+pipeline:
+  mangle:
+    type: logstatus
+`)
+
+func ConfigFromJSON(t *testing.T, content []byte) *viper.Viper {
+	cfg := viper.New()
+	cfg.SetConfigType("yaml")
+	r := bytes.NewBuffer(content)
+	if err := cfg.ReadConfig(r); err != nil {
+		t.Fatalf("Failed to read config: %s", err)
+	}
+	return cfg
+}
+
+func getMangleLogStatus(t *testing.T) core.Mangler {
+	cfg := ConfigFromJSON(t, logStatusConfig)
+	mangler, err := NewMangleLogStatus(cfg)
```

Comment:
`NewMangleLogStatus` doesn't use the `cfg` argument, so you can simply pass a `viper.New()` as argument and there's no need for `logStatusConfig` and `ConfigFromJSON`.